src/components/Layout/Header/GlobalSearch/GlobalSearchButton.tsx

```diff
@@ -73,14 +74,14 @@ export const GlobalSeachButton = () => {
   const [assetResults, stakingResults, lpResults, txResults] = results
   const flatResults = useMemo(() => [...results, sendResults].flat(), [results, sendResults])
   const resultsCount = flatResults.length
+  const isMac = /Mac|iPhone|iPod|iPad/.test(navigator.userAgent)
```

Comment:
Can probably keep `Mac` only as the user agent RegExp here since we're only displaying the keybindings on desktop now

Reply:
I think it will match them anyways because the UA for iPhone/iPad (didn't check iPod) almost always contain "like Mac OS" for some reason 😆 I don't think it will hurt to leave it like this.

Reply:
> I think it will match them anyways because the UA for iPhone/iPad (didn't check iPod) almost always contain "like Mac OS" for some reason 😆 I don't think it will hurt to leave it like this.

Precisely ☝🏽 eager regex on `Mac` will work just the same and match the `isMac` terminology. Won't change anything at runtime though, only OCD

Reply:
Ok, done ;)